bgpd/bgp_zebra.c

```diff
@@ -2705,6 +2706,35 @@ static int bgp_pbr_get_ifnumber(struct bgp *bgp)
 	return cnt;
 }
 
+struct egress_data {
+	uint32_t nexthop[30];
+	uint16_t cost[30];
+	uint32_t destination;
+};
+
+//process next-hop and cost and encode the data in the message 
+//send to zebra
+void bgp_send_infiot_egress(struct egress_data info, int size) {
+	struct stream *s;
+	int ret = 0;
+	s = zclient->obuf;
+	stream_reset(s);
+	zclient_create_header(s,
+			      TRUE ? ZEBRA_INFIOT_EGRESS_ADD :
+			      ZEBRA_INFIOT_EGRESS_DELETE,
+			      VRF_DEFAULT);
+	stream_putl(s, size);
+	stream_putl(s, info.destination);
+	for(int i=0;i<size;i++) {
```

Comment:
Minor request, coding convention, missing spaces Example. VSS Code or other IDEs fix this automatically 

for(int i=0;i<size;i++)      <- Current
for(int i=0; i<size; i++)    <- With Space

Reply:
Ack